### ISSUE TITLE
[BUG] converted subscription profile does not work on iOS due to the log config [ client: sing-box 1.7.0 (15) ]

### ISSUE THREAD

**Issue by IceCodeNew** [tindy2013/subconverter#664]
### 确认版本最新

- [X] 我已经确认在最新Action编译的版本中复现

### 检索issue

- [X] 我已经确认之前没有issue涉及此BUG

### subconverter版本

8b45e06d0d1aa004b972d06d5f2ca917cddf2000

### 转换过程

clash to sing-box

### 转换设置

https://raw.githubusercontent.com/tindy2013/subconverter/master/base/base/singbox.json
```
{
  "log": {
    "disabled": false,
    "level": "info",
    "output": "box.log",
    "timestamp": true
  },
  "dns": {},
  "ntp": {
    "enabled": false,
    "server": "time.apple.com",
    "server_port": 123,
    "interval": "30m"
  },
  "inbounds": [
    {
      "type": "socks",
      "tag": "socks-in",
      "listen": "127.0.0.1",
      "listen_port": 2080
    }
  ],
  "outbounds": [],
  "route": {},
  "experimental": {}
}
```

### 复现步骤

`https://example.com/sub?target=singbox&url=[xxx]&emoji=true&list=false&tfo=false&scv=false&fdn=false&sort=false`

### 期望结果

Enable log by default, but output to the stdout, not a file.

### 实际结果

iOS client failed to create/open the file path of log file.

**Comment by IceCodeNew** [comment]
<img width="800" alt="image" src="https://github.com/tindy2013/subconverter/assets/32576256/e825f021-a5ff-4739-ae91-15af4bd05f07">

**Comment by tindy2013** [comment]
Fixed in 51b65ec
